packages/components/inputs/select-input/src/select-input.tsx

```diff
@@ -461,6 +461,7 @@ const SelectInput = (props: TSelectInputProps) => {
             } as ReactSelectProps['components']
           }
           menuIsOpen={props.isReadOnly ? false : props.menuIsOpen}
+          hideSelectedOptions={false}
```

Comment:
Why was this added?

Reply:
to not displaythe selected value(s) in the search box of the input, they should only display in the `TriggerButton`

Reply:
updating inputs is a whole other side to this, there will need to be a lot done in all the `select` inputs in particular.  But that is a separate scope of work.

Reply:
Yeah, this was just a flag for me because it's at the input level, which means it affects all input's, and by setting it to false, it's not configurable at all. Just wasn't sure if this had side effects outside filters that would be undesired, but sounds like there is more to figure out here.